### PR TITLE
Users APIs input and response structure update

### PR DESCRIPTION
**Pull request by anshulrr**
1. roles API  response structure update
2. update_user API input structure update
3. test and docs update

### CODE REVIEW

File: api.docs/includes/_users.md [hunk 1]
```diff
@@ -215,12 +230,12 @@ mutation updateUser($id: ID!, $input: UserInput!, $groupIds: [ID]!) {
     "name": "Updated Name",
     "roles": [
       "admin"
+    ],
+    "groupIds": [
+      1,
```
Comment (pankaj-ag) [inline api.docs/includes/_users.md:235]: I think this should be Gid not ID

File: lib/glific_web/schema/users_types.ex [hunk 2]
```diff
@@ -25,6 +25,11 @@ defmodule GlificWeb.Schema.UserTypes do
     end
   end
 
+  object :role do
+    field :id, :string
```
Comment (pankaj-ag) [inline lib/glific_web/schema/users_types.ex]: I think the type of this should be id not string.

Comment (anshulrr) [reply lib/glific_web/schema/users_types.ex]: We don't have ids for roles right now.
Updated this response structure after discussing it with the frontend team.

File: api.docs/includes/_users.md [hunk 3]
```diff
@@ -14,10 +17,22 @@ query {
...
+      {
+        "id": 1,
+        "label": "none"
+      },
+      {
+        "id": 2,
+        "label": "staff"
+      },
+      {
+        "id": 3,,
```
Comment (pankaj-ag) [inline api.docs/includes/_users.md]: Just a minor thing. Extra ,

**Pull request merged**